django/forms/widgets.py

```diff
@@ -875,10 +875,10 @@ class SplitDateTimeWidget(MultiWidget):
     supports_microseconds = False
     template_name = 'django/forms/widgets/splitdatetime.html'
 
-    def __init__(self, attrs=None, date_format=None, time_format=None):
+    def __init__(self, attrs=None, date_format=None, time_format=None, date_attrs=None, time_attrs=None):
```

Comment:
Others attributes of the `SplitHiddenDateTimeWidget` constructor aren't described in documentation that referenced to the `SplitDateTimeWidget` class, i.e.:
```Similar to SplitDateTimeWidget, but uses HiddenInput for both date and time.```
 I don't think that we should replicate documentation in this case.

I added release notes.